cmd/sonobuoy/app/gen.go

```diff
@@ -279,7 +278,7 @@ func (g *genFlags) resolveConfig() *config.Config {
 		// Set these values as if the user had requested the defaults.
 		if g.genflags != nil {
 			for _, v := range conf.PluginSelections {
-				g.genflags.Lookup("plugin").Value.Set(v.Name)
+				g.genflags.Lookup("plugin").Value.Set(v.Name) //nolint:errcheck
```

Comment:
`//nolint:errcheck` will tell `golangci-lint` to not mark this as `errcheck` lint failure. the only resolutions I can think of are:
```go
// practically silencing error
_ = g.genflags.Lookup("plugin").Value.Set(v.Name)
```
or 
```go
// potentially changing behavior, because previously we ignored this error if they exist
err := g.genflags.Lookup("plugin").Value.Set(v.Name)
if err != nil {
	...
}
```

is there other alternatives?

Reply:
I think there may be one more option.  Looking at the doc for `errcheck` linter, it supports an external file where you can put package, functions, methods to skip.

Take a look here https://github.com/kisielk/errcheck#excluding-functions